docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+you must individually connect with it through the Control Center using the steps below
+
+#. Navigate to the Advanced tab
+#. Find Module ID, and change it to a unique value from all other modules you would like to connect with
+
+.. note::
+    In order to communicate with multiple modules at once, they all must have the same configured serial baud rate. If they do not, you will 
+    only be able to communicate with those modules with a baud rate equal to Serial Baud Rate.
+
+Module Connection Demonstration
+====================================
+An example of connecting with 4 Vertiq modules is presented in the screen capture below. In this example, 4 Vertiq 23-06 2200Kv 
+modules are connected, each with a different Module ID (0, 1, 2, and 3), configured to use a 921600 baud rate.
+
+.. raw:: html
+
+    <style type="text/css">
+    .center_vid {   margin-left: auto;
+                    margin-right: auto;
+                    display: block;
+                    width: 75%; 
+                }
+    </style>
+    <video class='center_vid' controls><source src="../_static/control_center_pics/connection_screen_cap.mp4" type="video/mp4"></video>
+
+Connection Configuration in the GUI
+========================================
+The serial configuration used to attempt connection with your modules is presented in the top right corner of the Control Center:
+
+.. image:: ../_static/control_center_pics/serial_options.png
+
+Serial Baud Rate
+--------------------
+This is the baud rate with which we will open the serial port being used to communicate with the connected modules. In order for 
+communication to succeed, this value must be the same as the baud rate set on your module. By default, all Vertiq modules use a serial baud rate of 115200.
+
+Serial Port
+--------------------
+This is the serial port where you have connected your USB-to-UART device. After connecting your device, you will see an option in the dropdown appear. 
+Click it to select it.
+
+The names that appear in this box are OS dependent. For example, on Linux, you will see devices with the name “/dev/ttySX” where X is the value of the port.
+
+Connect Button
+--------------------
+
+Once you have selected your target baud rate and your USB-to-UART device's serial port, you are ready to connect with your module. 
+
+Connect your module to the USB-to-UART device as described :ref:`above <connection_guide>`, and power on your module. 
+
+Click the connect button in order to open serial communication between the Control Center and your module. 
+
+If you successfully found and communicated with a module, you will see its information appear in the :ref:`Information section <information_box>` in the bottom left. 
+You will also see the :ref:`Module Connections <module_connections>` box list the number of detected modules as non-zero. 
+
+The CONNECT button now presents DISCONNECT.
```

Comment:
The phrasing is a bit odd here. Maybe "The CONNECT button will now change to the DISCONNECT button"